src/psij/web/js/BuildAccordianModelFromSites.js

```diff
@@ -0,0 +1,96 @@
+/*
+    This builds the DataModel for the Calendar View.
+    It preps the Data for consumption.
+ */
+PS.BuildAccordianModelFromSites = function() {
+
+    var get_states_ = function( months ) {
```

Comment:
Well, we need to maintain this code, and having consistent styling and naming conventions help with that so that, in the long run, there is less effort in maintaining the product so that effort can be focused on other things, such as features.

Reply:
Agreed.  from now on, I'll try to remember no spaces after or before a ( ).

Reply:
Should fix in the future: #42.